content/tutorials/history.textile

```diff
@@ -33,6 +33,56 @@ You have now enabled history for all channels in the @persisted@ namespace i.e.
 
 h2. Step 3 - Install Ably
 
+blang[java].
+  To start using Ably in your Android app, you need to include the Ably Client library. We recommend that you include the latest client library via Gradle in your module-level @gradle.build@ file.
+
+  ```[groovy]
+  apply plugin: 'com.android.application'
+  ...
+  dependencies {
+      ...
+      compile 'io.ably:ably-java:0.8.3'
+  }
+  ```
+
+  Also add following repo for a sub-dependency in your project-level @gradle.build@ file.
+  
+  ```[groovy]
+  allprojects {
+    repositories {
+        ...
+        maven { url "https://raw.github.com/paddybyers/Java-WebSocket/mvn-repo/" }
+    }
+  }
+  ```
+  
+  After you add neccessary dependencies, you can import AblyRest class into your code and initialize it.
+
+  ```[java]
+  import io.ably.lib.rest.AblyRest;
+
+  public class ExampleActivity extends AppCompatActivity {
+    private final static String API_KEY = "INSERT-YOU-API-KEY-HERE"; 
```

Comment:
Also, we need to raise an exception if that API key has not been set with a suitable error message shown to the user, like we do with other demos, see https://github.com/ably/tutorials/blob/history-ruby/example.rb#L4